BGL/doc/BGL/Concepts/HalfedgeGraph.h

```diff
@@ -55,5 +55,11 @@ Expression                              | Returns
 `next(h, g)`                            | `halfedge_descriptor`                                                        | The next halfedge around its face.
 `prev(h, g)`                            | `halfedge_descriptor`                                                        | The previous halfedge around its face.
 `boost::graph_traits<G>::%null_halfedge()` | `halfedge_descriptor`                                                     | Returns a special halfedge that is not equal to any other halfedge.
+
+\cgalHeading{Invariants}
+
+`halfedge(edge(h,g))==h`
```

Comment:
No, the invariant does not impose the same thing. It imposes that the source and target of an edge won't be swapt.